### PR TITLE
reactjs-component: Updating to version v0.16.0-beta.17

### PR DESCRIPTION
**Pull request by mlunoe**
PRs in this update:
https://github.com/mesosphere/reactjs-components/pull/399
https://github.com/mesosphere/reactjs-components/pull/400
https://github.com/mesosphere/reactjs-components/pull/398

:rotating_light: Blocks this PR: https://github.com/dcos/dcos-ui/pull/2078

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by mlunoe** [comment]
Run integration tests

File: src/js/utils/__tests__/Util-test.js [hunk 1]
```diff
@@ -183,40 +183,6 @@ describe('Util', function () {
...
-      jest.runAllTimers();
-
-      // The calls should be two because #throttle will remember if it
-      // was called during the wait and will invoke itself immediately once the
-      // wait is over.
-      expect(func.mock.calls.length).toBe(2);
-    });
-
-  });
-
```
Comment (jfurrow) [inline src/js/utils/__tests__/Util-test.js]: 🎉

**Comment by jfurrow** [comment]
Works as expected! Good stuff! :ship:

**Comment by mlunoe** [comment]
Run integration tests

**Comment by mlunoe** [comment]
Assigning to you Mat, since the other guys are on Holiday

**Pull request merged**